lib/parse_cmds.js

```diff
@@ -0,0 +1,543 @@
+// need to set all these locally in function scope
```

Comment:
It still runs just fine, these can be changed later when tests are added for each function, I just noted it for someone to do later.